Source/Libraries/FaultData/DataOperations/ConfigurationOperation.cs

```diff
@@ -461,11 +461,13 @@ string GetAggregatePrefix(double multiplier)
 
             Func<SourceIndex, bool> sourceIndexFilter = GetSourceIndexFilter(appDataType);
 
+            IEnumerable<object> tmp = sourceIndexMap
+                .SelectMany(item => Translate(item.SourceIndex), (item, SourceIndex) => new { item.Series, SourceIndex });
+
```

Comment:
Looks like code you added for debugging?

```suggestion
```

Reply:
Yes, I missed this one. Thank you for catching that